.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: clojure
-lein: 2.9.1
+dist: xenial
```

Comment:
To be honest, I don't think we realized that travis was still using Xenial by default. We primarily use Bionic in our internal enterprise product test suites and often use open source libraries to ensure that we work on newer platforms. If anything we want to move from the default to Bionic or Focal, not pin back to Xenial.

Reply:
removed distribution settings now  ,Apologies forgot to remove i used to test all the dist as well (boinic,focal ,xenial)